composer.json

```diff
@@ -14,14 +14,14 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
```

Comment:
I wouldn't say it is required, but http-cache v2.0 will set this constraint anyway. On the other hand, php 5.5 is EOL